packages/api/tests/endpoints/collections/update-collection.js

```diff
@@ -187,9 +224,21 @@ test('PUT replaces an existing collection', async (t) => {
     created_at: originalPgRecord.created_at,
     updated_at: actualPgCollection.updated_at,
   });
+
+  const { Messages } = await sqs().receiveMessage({
+    QueueUrl: t.context.QueueUrl,
+    WaitTimeSeconds: 10,
+  }).promise();
+
+  t.is(Messages.length, 1);
+
+  const message = JSON.parse(JSON.parse(Messages[0].Body).Message);
+  const omitFields = ['createdAt', 'updatedAt'];
+  t.is(message.event, 'Update');
+  t.deepEqual(omit(message.record, omitFields), omit(updatedCollection, omitFields));
```

Comment:
Again, I think we should compare against `actualCollection` and not `updatedCollection` given that what we want to prove is that the record in the SNS message matches what was written to the database

Reply:
[`754f871` (#2432)](https://github.com/nasa/cumulus/pull/2432/commits/754f871e6b2a2231b24c1dc75a24f09bf623fabd)